### PR TITLE
Fix LineLength offenses in /test

### PR DESCRIPTION
**Pull request by JoeCohen**

### CODE REVIEW

**Comment by JoeCohen** [comment]
I reviewed this for possible removal of extra spaces (other than those required by RuboCop) which caused LineLength to be > 80.

File: test/general_extensions.rb [hunk 1]
```diff
@@ -460,9 +464,11 @@ def assert_string_equal_file(str, *files)
         result = true
         break
       elsif !msg
-        # Write out expected (old) and received (new) files for debugging purposes.
-        File.open(filename + ".old", "w:#{encoding}") { |fh| fh.write(template) }
-        File.open(filename + ".new", "w:#{encoding}") { |fh| fh.write(str) }
+        # Write out expected (old) and received (new) files for debugging.
+        File.
+          open(filename + ".old", "w:#{encoding}") { |fh| fh.write(template) }
```
Comment (mo-nathan) [inline test/general_extensions.rb]: I'd change these to
```
... do |fh|
  ...
end
```

**Comment by mo-nathan** [comment]
One minor comment.

File: test/general_extensions.rb (hunk 1 above)
Comment (JoeCohen) [reply test/general_extensions.rb]: Yes, that makes more sense. Will fix.

**Pull request merged**